### PR TITLE
Feature/add checks

### PR DESCRIPTION
**Pull request by AdrienR**
Hi,

Thank for your probe, but i also need to check raid array health, so their is my additions.

### CODE REVIEW

**Comment by martialblog** [comment]
Hi, Thanks. I will take a look at it.

File: check_disk_btrfs [hunk 1]
```diff
@@ -207,6 +229,19 @@ def main():
 
         perfdata.append(perf)
 
+    if args.missing:
+        is_device_missing = check_missing_device(args.volume, args.timeout, args.sudo)
+    
+        if is_device_missing is True:
```
Comment (martialblog) [inline check_disk_btrfs]: This line can simply be `if is_device_missing:` the is True conditional is redundant. 

And there's some whitespaces in line 234 that can be removed

File: check_disk_btrfs [hunk 2]
```diff
@@ -207,6 +229,19 @@ def main():
...
+            out = "A device is missing in volume %s" % args.volume
+            crit.append(out)
+
+    if args.error:
+        scrub_error = check_scrub_error(args.volume, args.timeout, args.sudo)
+        if scrub_error:
```
Comment (martialblog) [inline check_disk_btrfs:246]: Can we rename that function "has_scrub_error"? Then the condition reads nicer. Just a minor change.

**Review by martialblog — CHANGES_REQUESTED** [review]
Looks good. Just some minor changes.

Thanks for writing tests!

File: check_disk_btrfs (hunk 2 above)
Comment (AdrienR) [reply check_disk_btrfs:246]: Yep, it's done

File: check_disk_btrfs (hunk 1 above)
Comment (AdrienR) [reply check_disk_btrfs]: I think it's more readable by putting is True, so you know you are checking a bool, not a None or something

File: check_disk_btrfs (hunk 2 above)
Comment (martialblog) [reply check_disk_btrfs:246]: Thanks

**Comment by martialblog** [comment]
@AdrienR could you rebase the branch and resolve the conflicts? Then I will merge this

**Comment by AdrienR** [comment]
> 

Thank, should be good now.

**Pull request merged**